src/NextjsBuild.ts

```diff
@@ -13,6 +13,7 @@ const NEXTJS_STATIC_DIR = 'static';
 const NEXTJS_PUBLIC_DIR = 'public';
 const NEXTJS_BUILD_STANDALONE_DIR = 'standalone';
 const NEXTJS_BUILD_STANDALONE_ENV = 'NEXT_PRIVATE_STANDALONE';
+const IMAGE_MANIFEST_FILE = 'next.config.js';
```

Comment:
looks odd

Reply:
Yea, I'll look into reading the `required-server-files.json manifest` tomorrow.